crates/blockchain/blockchain.rs

```diff
@@ -206,5 +307,94 @@ fn get_total_blob_gas(tx: &EIP4844Transaction) -> u64 {
     GAS_PER_BLOB * tx.blob_versioned_hashes.len() as u64
 }
 
+fn total_difficulty_check<'a>(
```

Comment:
is this necesary? post-merge the total difficulty doesn't change. Pre-merge FCU doesn't exist.